src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -998,6 +998,20 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 		window?.win?.webContents.toggleDevTools();
 	}
 
+	async openDevToolsWindow(windowId: number | undefined, url: string): Promise<void> {
+		const parentWindow = this.codeWindowById(windowId);
+		if (!parentWindow) {
+			return;
+		}
+		const options = this.instantiationService.invokeFunction(defaultBrowserWindowOptions, defaultWindowState(), { forceNativeTitlebar: true, hideBecauseShadowWindow: false });
+		options.backgroundColor = undefined;
+
+		const devToolsWindow = new BrowserWindow(options);
```

Comment:
@tmm1 @deepak1556 I was expecting some code similar to the GPU window in the method below that would close this window when the parent window closes?

```ts
parentWindow.win?.on('close', () => {
```